flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/GateBuffersSpecTest.java

```diff
@@ -143,17 +144,25 @@ void testCalculationWithConfiguredZeroExclusiveBuffer(ResultPartitionType partit
     void testConfiguredMaxRequiredBuffersPerGate(ResultPartitionType partitionType) {
         Optional<Integer> emptyConfig = Optional.empty();
         int effectiveMaxRequiredBuffers =
-                getEffectiveMaxRequiredBuffersPerGate(partitionType, emptyConfig);
+                getEffectiveMaxRequiredBuffersPerGate(partitionType, emptyConfig, false);
         int expectEffectiveMaxRequiredBuffers =
                 isPipelinedOrHybridResultPartition(partitionType)
                         ? InputGateSpecUtils.DEFAULT_MAX_REQUIRED_BUFFERS_PER_GATE_FOR_STREAM
-                        : InputGateSpecUtils.DEFAULT_MAX_REQUIRED_BUFFERS_PER_GATE_FOR_BATCH;
+                        : DEFAULT_MAX_REQUIRED_BUFFERS_PER_GATE_FOR_BATCH;
```

Comment:
Why `DEFAULT_MAX_REQUIRED_BUFFERS_PER_GATE_FOR_BATCH` is different with `InputGateSpecUtils.DEFAULT_MAX_REQUIRED_BUFFERS_PER_GATE_FOR_STREAM`. 

We'd better keep the two import formats consistent.